packages/core/src/transport/eventBridge.ts

```diff
@@ -5,21 +5,31 @@ export interface BrowserWindow extends Window {
 }
 
 export interface DatadogEventBridge {
+  getAllowedWebViewHosts(): string
   send(msg: string): void
 }
 
 export function getEventBridge<T, E>() {
-  const datadogEventBridge = getEventBridgeGlobal()
+  const eventBridgeGlobal = getEventBridgeGlobal()
+
+  if (!eventBridgeGlobal) {
+    return { getAllowedWebViewHosts: () => [], send: () => undefined }
+  }
```

Comment:
Do we need to return a fake event bridge here? Why not returning `undefined`?

Reply:
Sure it can work.